tools/jit/gen_jit_dispatch.py

```diff
@@ -304,12 +259,12 @@ def emit_decl(decl):
         all_real_arguments_are_inputs = tuple(arg['simple_type'] not in default_only_types for arg in arguments)
         only_tensors_are_inputs = tuple(is_tensor_arg(arg) for arg in arguments)
 
-        variants = [emit_decl_variant(decl, all_real_arguments_are_inputs, has_tensorlist)]
+        variants = [emit_decl_variant(decl, all_real_arguments_are_inputs)]
         # in some cases there are no inputs that are possibly attributes, so the
         # variants are actually the same. If so avoid generating both to save compilation
         # time.
         if all_real_arguments_are_inputs != only_tensors_are_inputs:
-            variants += [',', emit_decl_variant(decl, only_tensors_are_inputs, has_tensorlist)]
+            variants += [',', emit_decl_variant(decl, only_tensors_are_inputs)]
```

Comment:
aside: I guess we can clean this up as well now that nothing uses the attribute versions

Reply:
Yes, there are many cleanups we can do now. I just didn't have time, and chose to now add irrelevant changes to this diff./